docs/template_plugin/tests/functional/op_reference/ti.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <tuple>
+#include <openvino/opsets/opset8.hpp>
+
+#include "base_reference_test.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+
+using namespace reference_tests;
+
+class ReferenceTensorIteratorTest : public testing::Test, public CommonReferenceTest {
+public:
+    void SetUp() override {
+        SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    }
+
+public:
```

Comment:
Why?

Reply:
Removed, added support of parameters, like for 'If' operation